OpenRA.Game/Network/OrderIO.cs

```diff
@@ -33,7 +33,10 @@ public OrderPacket(MemoryStream data)
 
 		public IEnumerable<Order> GetOrders(World world)
 		{
-			return orders ?? ParseData(world);
+			if (orders == null)
```

Comment:
```suggestion
			// When using local orders we need to make sure they are refreshed, to match the deserialised orders from other clients
			if (orders == null)
```

Perhaps add a comment here

Reply:
added.